examples/iDynTreeWrappers/visualizeRobot.m

```diff
@@ -56,53 +65,59 @@
     'neck_roll';
     'neck_yaw'
     };
-%
+
+% Main variable of iDyntreeWrappers used for many things including updating
+% robot position and getting world to frame transforms
 KinDynModel = iDynTreeWrappers.loadReducedModel(jointOrder,'root_link',modelPath,fileName,false);
 
 % create vector of positions
 joints_positions=zeros(KinDynModel.NDOF,1);
 
-% Prepare figure, handles and variables required for the update
-[transforms,linkNames,linkNames_idyn,map,linkMeshInfo,meshHandles,parent,mainHandler]=iDynTreeWrappers.prepareVisualization(KinDynModel,meshFilePrefix);
-% Note: For default visualization these variables can be ignored:
-%       map,linkMeshInfo,meshHandles,parent,mainHandler.
+fakeWorld=[1,0,0,0;0,1,0,0;0,0,1,0.6;0,0,0,1];%eye(4);
+                
```

Comment:
Please avoidi commiting trailing whitespaces. 
```suggestion

```

Reply:
I do not notice it sometimes, is Matlab able to do this automatically? or Github?

Reply:
> 
> 
> I do not notice it sometimes, is Matlab able to do this automatically? or Github?

I don't know.

Reply:
For now, I opened in atom and saved without trailing whitespaces but I do not think this is sustainable in the long wrong since most of not all editing of .m files happens in matlab